README.md

```diff
@@ -1 +1,93 @@
-# nodeChainMiddleware
\ No newline at end of file
+# Node Chain Middleware
+
+*Middleware are like links in chain.*
+
+[![Build Status][travis-image]][Travis CI]
+[![Dependencies][dependencies-image]][Dependencies]
+[![Dev Dependencies][devdependencies-image]][Dev Dependencies]
+[![codecov.io][codecov-image]][Code Coverage]
+
+About
+-----
+
+This makes it easier to use middleware together by "chaining" them to be single function which can then be called.
```

Comment:
I think a bit more here would be good.  Suggestion:

> This module combines multiple middlewares together and "chains" them into a single function.  Unlike normal
> middleware additions, you are able to merge arrays of middleware and even nested arrays of middleware.  They
> all act as you would expect.
>
> This module is currently in use for [Restify] and should work for [Express] and similar servers that use the same
> pattern for middleware.

Then just link up Restify and Express.